fe/src/main/java/org/apache/doris/transaction/GlobalTransactionMgr.java

```diff
@@ -818,24 +817,26 @@ public boolean isPreviousTransactionsFinished(long endTransactionId, long dbId,
         return true;
     }
 
-    // check if there exists a intersection between the source tableId list and target tableId list
-    // if one of them is null or empty, that means that we don't know related tables in tableList,
-    // we think the two lists may have intersection for right ordered txns
+    /**
+     * check if there exists a intersection between the source tableId list and target tableId list
+     * if one of them is null or empty, that means that we don't know related tables in tableList,
+     * we think the two lists may have intersection for right ordered txns
+     */
     public boolean isIntersectionNotEmpty(List<Long> sourceTableIdList, List<Long> targetTableIdList) {
         if (CollectionUtils.isEmpty(sourceTableIdList) || CollectionUtils.isEmpty(targetTableIdList)) {
             return true;
         }
-        for (int i = 0; i < sourceTableIdList.size(); i++) {
-            for (int j = 0; j < targetTableIdList.size(); j++) {
-                if (sourceTableIdList.get(i).equals(targetTableIdList.get(j))) {
+        for (Long aLong : sourceTableIdList) {
```

Comment:
Could  rename `aLong` and `value ` to `src` and `target`

Reply:
done